lib/features/children/details/pages/child_details_page.dart

```diff
@@ -172,7 +172,17 @@ class ChildDetailsPage extends StatelessWidget {
                                   .state
                                   .user
                                   .isMissingcardDetails) {
-                                CreditCardDetails.show(context);
+                                CreditCardDetails.show(
+                                  context,
+                                  onClose: () {
+                                    _navigateToEditAllowanceScreen(
```

Comment:
Is this onClose correct? Now if you just close the creditcard details rather than entering your details, does this get triggered as well? Is that what we want?

Reply:
I renamed it, its triggered after registration success